docs/admin/rolling-upgrade.txt

```diff
@@ -137,6 +137,11 @@ need to send the ``USR2`` signal::
   If your system uses SysVinit (e.g. Debian Wheezy), you can also use ``service
   crate graceful-stop``.
 
+.. WARNING::
+
+    Stopping a crate process using the ``USR2`` signal is deprecated
+    and will be removed in a future release.
```

Comment:
I think this should link back to the decomission statement. Or change the example altogether to use the decomission statement and mention the `USR2` in a note.

The deprecation should also be mentioned in the release notes.